modules/apps/opensocial/opensocial-portlet/docroot/WEB-INF/src/com/liferay/opensocial/shindig/servlet/ShindigFilter.java

```diff
@@ -18,23 +18,7 @@
 
 import com.liferay.opensocial.shindig.util.HttpServletRequestThreadLocal;
 import com.liferay.opensocial.shindig.util.ShindigUtil;
-import com.liferay.petra.encryptor.Encryptor;
```

Comment:
Hi @dantewang @ChiZelin ,
This removal is not completed. You need also to remove the property from portal.properties, or else this will leave an unread property. While removing the whole property meaning this whole feature is removed, which is not we can not decided from engineering point of view. So let's keep this untouched, and wait for project manager's request to remove it.

Tina.